schemas/account/createAccount.json

```diff
@@ -1,6 +1,28 @@
 {
     "components": {
         "schemas": {
+            "CreateCustomerRelationships": {
+                "allOf": [
+                    {
+                        "type": "object",
+                        "required": ["customer"],
+                        "additionalProperties": false
+                    },
+                    {
+                        "$ref": "../relationships.json#/components/schemas/CustomerRelationship"
+                    }
+                ]
+            },
+            "CreateCreditAccountRelationships": {
+                "allOf": [
+                    {
+                        "title": "Create Push To Card Payment Relationships"
+                    },
+                    {
+                        "$ref": "#/components/schemas/CreateCustomerRelationships"
+                    }
+                ]
+            },
```

Comment:
you don't need to split those declarations, having two allOfs under CreateCreditAccountRelationships will work

Reply:
fixed